.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: cpp
+compiler:
+  - gcc
+  - clang
+
+before_install:
+  - if [ "${TRAVIS_OS_NAME}" = "linux" ]; then
+         wget http://www.cmake.org/files/v3.3/cmake-3.3.0-Linux-i386.tar.gz
+      && tar -xzf cmake-3.3.0-Linux-i386.tar.gz
+      && sudo cp -fR cmake-3.3.0-Linux-i386/* /usr
+      && sudo apt-add-repository -y ppa:ubuntu-toolchain-r/test
+      && sudo apt-get -qq update
+      && sudo apt-get -qq install g++-6 libc6-i386
+      && export CXX="g++-6"
+      && export CC="gcc-6"
```

Comment:
I pulled the bulk of this script from a much older project.  At the time the default version of GCC may have been 4.7 and didnt support all of C++ 11 if I remember correctly (Since it was based on 12.04 or earlier).  I didn’t check what the default GCC is on Travis now, but should do that.